packages/@lwc/engine/dom/src/apis/build-custom-element-constructor.ts

```diff
@@ -7,15 +7,14 @@
 
 import { create, isUndefined, keys } from '@lwc/shared';
 
-import { LightningElement } from './lightning-element';
-
 import {
     createVM,
     connectRootElement,
     disconnectRootElement,
     getAttrNameFromPropName,
     getComponentInternalDef,
     isAttributeLocked,
+    LightningElement,
```

Comment:
What I was asking is, how come we are making changes to import LightningElement from engine-core instead of from within the same engine-dom package? I’m not clear what the difference is.

Reply:
Now I see, we are importing `LightningElement` just to get the interface here, not the constructor. The `LightningElement` interface is dictated by engine-core, while engine-dom is patching the constructor without changing its original interface.